cpp/test/Ice/ami/Test.ice

```diff
@@ -41,7 +34,8 @@ interface TestIntf
                     out int eight, out int nine, out int ten, out int eleven);
     int opBatchCount();
     bool waitForBatch(int count);
-    void close(CloseMode mode);
+    void close();
```

Comment:
`closeConnection` and `abortConnection` would be clearer.

Reply:
Fine
I added the `Connection` suffix to the operation names.